lib/helpers/calculator_helpers.js

```diff
@@ -495,4 +481,84 @@ module.exports = class CalculatorHelpers {
     };
     return elmFunction;
   }
+
+  static map;
+  static systemUriByName(systemName) {
+    if(this.map === undefined) {
+      /* eslint dot-notation:0 */
+      this.map = {};
+      this.map['medication-statement-status'] = 'http://hl7.org/fhir/CodeSystem/medication-statement-status';
+      this.map['medicationrequest-intent'] = 'http://hl7.org/fhir/CodeSystem/medicationrequest-intent';
+      this.map['medicationrequest-status'] = 'http://hl7.org/fhir/CodeSystem/medicationrequest-status';
+      this.map['discharge-disposition'] = 'http://terminology.hl7.org/CodeSystem/discharge-disposition';
```

Comment:
Probably we don't need that map in cqm-execution if I merge another PR which updated ValueSets with system URIs